sass/directives/_visual-guides.scss

```diff
@@ -125,7 +118,7 @@ $visual-guide-breakpoint-small: 750px;
   }
 
   // The step must be within a <span>, not a <div>, and must contain another <span> for the actual counter
-  span.visual-guide__step {
+  div.visual-guide__step {
```

Comment:
Typically, we don't bother with this level of specificity - it reduces the portability/re-usability of this code, requiring that any use of this class be applied to a div. I think our code can make the safe assumption that if you're using the class, you want its styles in every case for any kind of element. Also, your comment above says the step /must/ be within a `span`. Besides that comment needing updated to say `div`, is any of this truly required?

If we end up deciding to change this, there are other places in the Overview code that could incorporate this change as well.